db/dcrsqlite/sqlite.go

```diff
@@ -79,29 +80,31 @@ func NewDB(db *sql.DB) (*DB, error) {
 	}
 
 	// Ticket pool queries
-	d.getPoolSQL = fmt.Sprintf(`select hash, poolsize, poolval, poolavg, winners`+
-		` from %s where height = ?`, TableNameSummaries)
-	d.getPoolByHashSQL = fmt.Sprintf(`select height, poolsize, poolval, poolavg, winners`+
-		` from %s where hash = ?`, TableNameSummaries)
-	d.getPoolRangeSQL = fmt.Sprintf(`select height, hash, poolsize, poolval, poolavg, winners `+
-		`from %s where height between ? and ?`, TableNameSummaries)
-	d.getPoolValSizeRangeSQL = fmt.Sprintf(`select poolsize, poolval `+
-		`from %s where height between ? and ?`, TableNameSummaries)
-	d.getAllPoolValSize = fmt.Sprintf(`select distinct poolsize, poolval, time `+
-		`from %s order by time`, TableNameSummaries)
-	d.getWinnersSQL = fmt.Sprintf(`select hash, winners from %s where height = ?`,
+	d.getPoolSQL = fmt.Sprintf(`SELECT hash, poolsize, poolval, poolavg, winners`+
+		` FROM %s WHERE height = ?`, TableNameSummaries)
+	d.getPoolByHashSQL = fmt.Sprintf(`SELECT height, poolsize, poolval, poolavg, winners`+
+		` FROM %s WHERE hash = ?`, TableNameSummaries)
+	d.getPoolRangeSQL = fmt.Sprintf(`SELECT height, hash, poolsize, poolval, poolavg, winners `+
+		`FROM %s WHERE height BETWEEN ? AND ?`, TableNameSummaries)
+	d.getPoolValSizeRangeSQL = fmt.Sprintf(`SELECT poolsize, poolval `+
+		`FROM %s WHERE height BETWEEN ? AND ?`, TableNameSummaries)
+	d.getAllPoolValSize = fmt.Sprintf(`SELECT distinct poolsize, poolval, time `+
+		`FROM %s ORDER BY time`, TableNameSummaries)
+	d.getWinnersSQL = fmt.Sprintf(`SELECT hash, winners FROM %s WHERE height = ?`,
 		TableNameSummaries)
-	d.getWinnersByHashSQL = fmt.Sprintf(`select height, winners from %s where hash = ?`,
+	d.getWinnersByHashSQL = fmt.Sprintf(`SELECT height, winners FROM %s WHERE hash = ?`,
 		TableNameSummaries)
 
-	d.getSDiffSQL = fmt.Sprintf(`select sdiff, diff from %s where height = ?`,
+	d.getSDiffSQL = fmt.Sprintf(`SELECT sdiff FROM %s WHERE height = ?`,
 		TableNameSummaries)
-	d.getSDiffRangeSQL = fmt.Sprintf(`select sdiff from %s where height between ? and ?`,
+	d.getLast24hrsDifficulty = fmt.Sprintf(`SELECT diff FROM %s WHERE time >= ? ORDER BY time limit 1`,
```

Comment:
The name `getLast24hrsDifficulty` is in accurate and confusing since it's not 24 hours but the difficulty at or after the given time.

Reply:
Will replace it it `getDifficulty` Then add a comment